themes/bootstrap3/templates/combined/jump-links.phtml

```diff
@@ -1,8 +1,8 @@
-<div class="combined-jump-links">
+<div class="combined-jump-links" style="visibility: hidden;">
```

Comment:
Using visibility instead of display: none to avoid the layout jerk when it becomes visible.